department-of-veterans-affairs/caseflow
# Shalliburton/13944 video hearing alerts

Resolves #13944 

### Description
Changes cancellation emails to be sent right away and adds the success alert when the cancellation emails have been sent

### Acceptance Criteria
- [ ] User cannot change hearing type while cancellation emails are being sent
- [ ] User receives success alert when the emails have been sent

### Testing Plan
1. Select judge "BVAYELLOW": http://localhost:3000/test/users
1. Navigate to hearings:  http://localhost:3000/hearings/schedule
1. Choose a "Video" hearing
1. **IMPORTANT:** Make sure there is a hearing location selected before continuing, otherwise there will be an error sending the cancellation emails
1. Click the "Edit hearing details" link
1. Change the hearing type to "Virtual" and ensure the hearing type dropdown is then disabled
1. Wait for the success alert and change the hearing type back to "Video"
1. Confirm the hearing type dropdown is disabled and wait for the emails to be sent
1. Once the emails have been sent, confirm that you receive a success alert


### User Facing Changes

**Before**
![BeforeEmail](https://user-images.githubusercontent.com/61989526/80392112-b871a800-887c-11ea-8747-96a099017730.png)


**After**
![AfterEmailInProgress](https://user-images.githubusercontent.com/61989526/80391854-50bb5d00-887c-11ea-8df5-4660f40aa13c.png)
![AfterEmailSuccess](https://user-images.githubusercontent.com/61989526/80391855-50bb5d00-887c-11ea-99d1-8c88dc353414.png)
